src/vos/vos_io.c

```diff
@@ -1736,6 +1765,10 @@ update_cancel(struct vos_io_context *ioc)
 	/* Cancel NVMe reservations */
 	vos_publish_blocks(ioc->ic_cont, &ioc->ic_blk_exts, false,
 			   VOS_IOS_GENERIC);
+
+	/* Abort dedup entries */
+	vos_dedup_process(vos_cont2pool(ioc->ic_cont), &ioc->ic_dedup_entries,
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?